query/src/api/http/v1/status.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+use std::time::SystemTime;
+
+use poem::web::Data;
+use poem::web::Json;
+use poem::IntoResponse;
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::sessions::SessionManager;
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Debug)]
+pub struct Status {
```

Comment:
should we also cover tenant_id and cluster_id?  since those things update dynamically

Reply:
ok I will check it

Reply:
@ZhiHanZ 

> since those things update dynamically

do you mean reload_config,  so you can get  tenant_id and cluster_id through /v1/config?